### PR TITLE
More intuitive explantion of strings formatting

### PR DESCRIPTION
**Pull request by knight42**

### CODE REVIEW

File: src/libcollections/fmt.rs [hunk 1]
```diff
@@ -434,37 +434,31 @@
 //!    in this case, if one uses the format string `{<arg>:<spec>.*}`, then the `<arg>` part refers
 //!    to the *value* to print, and the `precision` must come in the input preceding `<arg>`.
 //!
-//! For example, these:
+//! For example, the following calls all print the same thing `Hello x is 0.01000`:
 //!
 //! ```
-//! // Hello {arg 0 (x)} is {arg 1 (0.01) with precision specified inline (5)}
+//! // Hello {arg 0 ("x")} is {arg 1 (0.01) with precision specified inline (5)}
```
Comment (steveklabnik) [inline src/libcollections/fmt.rs]: why did the quotes get added here?

**Comment by steveklabnik** [comment]
Thanks! Quick question about the quotes.

File: src/libcollections/fmt.rs (hunk 1 above)
Comment (knight42) [inline src/libcollections/fmt.rs]: In case people mistake it for a variable.

Comment (steveklabnik) [inline src/libcollections/fmt.rs]: So then the output on line 437 needs to be updated as well, right?

Comment (knight42) [inline src/libcollections/fmt.rs]: er...but the output is `Hello x is 0.01000`, as written on line 437.
Is it necessary to add extra quotes around the `x`?

Comment (steveklabnik) [inline src/libcollections/fmt.rs]: gah, sorry. I mis-read the diff here; I thought these were inside of format strings. I was just wrong.

**Comment by knight42** [comment]
Could this be merged now?

**Pull request merged**